src/ui/guides/01_how_to_vpn.json

```diff
@@ -0,0 +1,34 @@
+{
+  "id": "how_to_vpn",
+  "title": "The top uses and benefits of using Mozilla VPN",
+  "image": "qrc:/guides/01_how_to_vpn.svg",
+  "blocks": [
+    { "id": "c_1",
+      "type": "text",
+      "content": "A VPN (or virtual private network) creates a secure and private connection by making a “tunnel” between your device and the web. Here are some of the top things you can do with Mozilla VPN." },
+    { "id": "c_2",
+      "type": "title",
+      "content": "Safeguard your personal data & identity" },
```

Comment:
https://mana.mozilla.org/wiki/display/FIREFOX/Word+List#WordList-ampersand/&

Should VPN follow the same guidelines?

Reply:
Good point.